powsybl-parent/powsybl-parent-ws/pom.xml

```
@@ -68,8 +68,11 @@
             <column name="myuuid" type="UUID">
             still generates
             <modifyDataType columnName="myuuid" newDataType="bytea"/>
+
+            Switching to liquibase-hibernate.version 4.26.0, which includes resolving the issue where it attempted to connect to the database during changelog generation.
+            This connection issue has been corrected in version 4.26.0, along with potential additional bug fixes and enhancements.
         -->
-        <liquibase-hibernate.version>4.23.2</liquibase-hibernate.version>
+        <liquibase-hibernate.version>4.26.0</liquibase-hibernate.version>
```

```suggestion
        <liquibase-hibernate.version>4.27.0</liquibase-hibernate.version>
```
from the [releases](https://github.com/liquibase/liquibase-hibernate/releases), it's not the last version

---

This is the version that contains the fix we made on the liquibase-hibernate repo and also I avoid taking the latest version